README.md

```diff
@@ -1,44 +1,87 @@
-# Laravel Kinesis Logger
+# Laravel Monolog Kinesis Driver
 
 [![Packagist](https://img.shields.io/packagist/v/Pod-Point/laravel-monolog-kinesis.svg)](https://packagist.org/packages/pod-point/laravel-monolog-kinesis)
 
-Package to log Laravel application logs to a Kinesis stream.
+A simple package to forward Laravel application logs to a Kinesis stream.
 
 ## Installation
 
-Require the package in composer:
+Require the package with composer:
 
-Laravel < 6.0
 ```bash
-composer require pod-point/laravel-monolog-kinesis:^2.0
+composer require pod-point/laravel-monolog-kinesis
 ```
 
-Laravel 6.0+
-```bash
-composer require pod-point/laravel-monolog-kinesis:^3.0
-```
+For Laravel < 6.0 you can use `pod-point/laravel-monolog-kinesis:^2.0`.
 
-## Usage
+### Setting up the AWS Kinesis service
 
-Publish the config files:
+Add your AWS key ID, secret and default region to your `config/services.php`:
```

Comment:
Could potentially make this a top level file rather than insisting on config being inside `services.php` which apps might be using for other things. Or even better, allow customisation of the config file path

Reply:
Just realised you'd already responded to a similar comment of mine previously